docs/user-guide/datatypes.rst

```diff
@@ -0,0 +1,82 @@
+##########
+Data Types
+##########
+
+Felis has an internal type system that is used to define the data types of columns in a schema.
+These are mapped to the appropriate SQL data types when the schema is created in a database and the
+corresponding `VOTable <https://ivoa.net/documents/VOTable/>`__ data type when writing information to
+`TAP_SCHEMA <https://www.ivoa.net/documents/TAP/20190927/REC-TAP-1.1.html#tth_sEc4>`_.
+
+The following values are supported for the
+`datatype <../dev/internals/felis.datamodel.Column.html#felis.datamodel.Column.datatype>`_ field of a column
+object:
+
++-----------+----------------------------------+--------------------------------------------------------------------------------------------------+
+| Type      | Description                      | Notes                                                                                            |
++===========+==================================+==================================================================================================+
+| boolean   | boolean value                    |                                                                                                  |
++-----------+----------------------------------+--------------------------------------------------------------------------------------------------+
+| byte      | 8-bit signed integer             |                                                                                                  |
```

Comment:
We need a footnote on `byte` here and in the next table to explain the issue with signedness.

Reply:
Could you provide text for this? I'm not sure what you'd like to say about it.

Reply:
"The Felis type `byte` currently maps to signed 8-bit integral types in the supported database back ends.  VOTable, and therefore TAP_SCHEMA, only has an **unsigned** 8-bit type.  The Felis mapping is under review at this time; in the mean time, if you are intentionally storing signed byte values in a database with a Felis-derived schema, we recommend the use of a VOTable type override in Felis, `votable:datatype: short` to ensure that the signedness of the values is preserved in TAP output."